starters/solidjs-tanstack-tailwind/src/pages/Home.jsx

```diff
@@ -2,20 +2,25 @@ import { NavLink } from '@solidjs/router';
 
 const Home = () => {
   return (
-    <div class="flex flex-col gap-2 items-center text-blue-800 underline text-base">
-      <NavLink
-        href="/counter"
-        class="hover:text-blue-500 transition-colors delay-100"
-      >
-        See Counter example component
-      </NavLink>
-      <NavLink
-        href="/api-example"
-        class="hover:text-blue-500 transition-colors delay-100"
-      >
-        See API example component
-      </NavLink>
-    </div>
+    <>
+      <header class="flex justify-center items-center text-white my-5 mx-auto bg-blue-500  w-full lg:w-[75%] p-4 text-lg ">
+        SolidJs, Tanstack and Tailwind Starter kit
```

Comment:
I guess if we decided not to use Solid-query, we should remove `Tanstack` from the header?

Reply:
Yes, i was hoping we create ticket to change everything related to it, not just the header. 

What do you think?

Reply:
That sounds good to me